pkg/virt-operator/webhooks/kubevirt-update-admitter_test.go

```diff
@@ -64,4 +65,67 @@ var _ = Describe("Validating KubeVirtUpdate Admitter", func() {
 			},
 		}, 0),
 	)
+
+	table.DescribeTable("test validateWorkloadUpdateMethods", func(oldSpec, newSpec *v1.KubeVirtSpec, accept bool) {
+		causes := validateWorkloadUpdateMethods(oldSpec, newSpec)
+		if accept {
+			Expect(causes).To(BeEmpty())
+		} else {
+			Expect(causes).To(Not(BeEmpty()))
+		}
```

Comment:
I hate boolean input arguments. 

I would prefer if you passed directly the expectation to the table, or, even better (opinionated ... ) if you decoupled the tests into 2 tables: one that the causes are expected to be empty, another where the causes are expected to *not* be empty.

Reply:
Done, I actually split it into one table and one `It()` statement as there was only one reject case.